mitmproxy/tools/console/overlay.py

```diff
@@ -120,7 +128,12 @@ def keypress(self, size, key):
             self.callback(self.choices[self.walker.index])
             signals.pop_view_state.send(self)
             return
-        elif key == "esc":
+        elif key in self.shortcuts:
+            shortcut_index = self.shortcuts.index(key)
```

Comment:
This raises a ValueError for keys which are not in `self.shortcuts` I think?

I'd prefer a `choice_by_shortcut` function as in https://github.com/mitmproxy/mitmproxy/pull/2834#pullrequestreview-93365506, mainly because that can be tested.

Reply:
@mhils it won't raise ValueError, because we check, whether key is in `self.shortcuts` or not -> `elif key in self.shortcuts`. So `key` is always in `self.shortcuts`, when we call `self.shortcuts.index(key)`
Ok, I'll change it on `choice_by_shortcut` :)